engine/src/main/java/com/redhat/cloud/notifications/events/FromOpenBridgeHistoryFiller.java

```diff
@@ -103,23 +100,27 @@ public void execute() {
         }
 
         for (ProcessingError pe : items) {
+            try {
 
-            // We may have seen an item before, skip it
-            String historyId = pe.getHeaders().get(RHOSE_ORIGINAL_EVENT_ID_HEADER);
-            if (alreadyProcessed(historyId)) {
-                continue;
-            }
-
-            statelessSessionFactory.withSession(statelessSession -> {
-                Map<String, Object> decodedPayload = decodeItem(pe);
-                // TODO reinjectIf needed (?)
-                try {
-                    camelHistoryFillerHelper.updateHistoryItem(decodedPayload);
-                    messagesWithError.increment();
-                } catch (Exception e) {
-                    Log.warn("|  History update failed", e);
+                // We may have seen an item before, skip it
+                String historyId = pe.getHeaders().get(RHOSE_ORIGINAL_EVENT_ID_HEADER);
+                if (alreadyProcessed(historyId)) {
+                    continue;
                 }
-            });
+
+                statelessSessionFactory.withSession(statelessSession -> {
+                    Map<String, Object> decodedPayload = decodeItem(pe);
+                    // TODO reinjectIf needed (?)
+                    boolean updated = camelHistoryFillerHelper.updateHistoryItem(decodedPayload);
+                    if (updated) {
+                        messagesWithError.increment();
+                    } else {
+                        Log.infof("RHOSE notification history update failed because no record was found with [id=%s]", historyId);
```

Comment:
The old `Log.warn` instruction created noise in Sentry. `Log.info` should be enough.